### PR TITLE
#54 - Integration tests enabled

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #54
Enabled pull and push integration tests for Linux. Added puzzle for Windows tests.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#116](https://github.com/artipie/docker-adapter/issues/116) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/51773657-63d9-4415-881a-abfa5dd059cf); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/docker/http/DockerSliceITCase.java [hunk 1]
```diff
@@ -39,18 +39,23 @@
...
  *
+ * @todo #54:30min Support tests running on Windows platform
+ *  Docker client on Windows cannot pull Linux images.
+ *  This requires to select image for tests depending on `os.name` property
+ *  and redesign pull tests to push images first.
  * @since 0.2
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@DisabledIfSystemProperty(named = "os.name", matches = "Windows.*")
 @SuppressWarnings({"PMD.TestClassWithoutTestCases", "PMD.AvoidDuplicateLiterals"})
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/docker/http/DockerSliceITCase.java:59]: @olegmoz we should be able to remove `PMD.TestClassWithoutTestCases` suppression already.

Comment (olegmoz) [reply src/test/java/com/artipie/docker/http/DockerSliceITCase.java:59]: @carlosmiranda that's true, thank you for noticing

**Comment by olegmoz** [comment]
@carlosmiranda ping

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will find at least 3 issues during next CR, thanks

**Comment by carlosmiranda** [comment]
@sereshqua agreed, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**